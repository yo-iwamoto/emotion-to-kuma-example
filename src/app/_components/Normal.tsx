'use client';

import { css } from '@emotion/react';

export function Normal() {
  return <p css={styles.text}>Normal</p>;
}

const styles = {
  text: css`
    color: green;
    font-size: 40px;
  `,
};
