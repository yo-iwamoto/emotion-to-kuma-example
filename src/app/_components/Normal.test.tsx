import { Normal } from './Normal';
import { render } from '@testing-library/react';

describe('Normal', () => {
  it('renders unchanged', () => {
    const { asFragment } = render(<Normal />);

    expect(asFragment()).toMatchSnapshot();
  });

  it('has color style', () => {
    const { getByText } = render(<Normal />);

    expect(getByText('Normal')).toHaveStyleRule('color', 'green');
  });
});
