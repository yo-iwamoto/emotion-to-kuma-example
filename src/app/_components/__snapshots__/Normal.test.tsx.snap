// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Normal > renders unchanged 1`] = `
<DocumentFragment>
  .emotion-0 {
  color: green;
  font-size: 40px;
}

<p
    class="emotion-0"
  >
    Normal
  </p>
</DocumentFragment>
`;
